modules/ROOT/pages/release-notes.adoc

```diff
@@ -518,8 +518,31 @@ The notable changes are:
 
 ==== Fixes in Bonita Runtime (including Bonita Applications)
 
+* BPA-135 - [SSO] Refreshing a UIB app page when the session is expired triggers a redirect to login.jsp
+* BPA-144 - [SSO] OIDC access tokens refresh is failing when the token is opaque (non JWT)
 * RUNTIME-28 - The deletion of a running process instance leaves lines in the DOCUMENT table. +
 Before this fix, when a process instance was deleted, the documents associated with it were not deleted. As a consequence, this behavior creates orphan documents that are unused and not linked to any other object. Note that it was not the case for archived process instances where documents were correctly deleted. +
 If you have a lot of orphan documents in your database, you can use the latest version of the xref:ROOT:purge-tool.adoc#delete-orphan-document-content[Purge Tool] to remove them.
+* RUNTIME-44 - NPE when iterating on multiple BDM objects in event subprocess
+* RUNTIME-1786 - When updating an SCA application with profiles mapping, the log is deceitful
+* RUNTIME-1870 - REST API: JSON in the HTTP GET response cannot be parsed `/API/bpm/caseInfo/+{id}+`
+* RUNTIME-1885 - Doubtful queries on application_* tables each time we navigate to a page
+* RUNTIME-1889 - Super Admin App page is refreshed for each page switch
+* RUNTIME-1911 - Using tenantid column in index causing query performance degradation
+* RUNTIME-1919 - Email connector fails if it includes an attachment
+* RUNTIME-1923 - REST API extension Fails on Second Call on `context.apiClient.getDAO`
+* RUNTIME-1932 - Cancelled and Aborted process instances are not returned when using the REST API (bpm/archivedCase)
+* RUNTIME-1940 - Process instance connectors may be displayed in task details page
+* RUNTIME-1947 - Oracle/MS SQL Server Deadlock Occurring During Mass Task Completion/Cancellation
 
 ==== Fixes in Bonita Studio (including Bonita UI Designer)
+
+* STUDIO-4477 - Some reserved keywords (limit) are missing during validation on Business Data Attributes
+* STUDIO-4516 - Errors at .bos file import and migration
+* STUDIO-4520 - Old version Git repository clone fails with NullPointerException during the execution of bonita-project-maven-plugin:create-bdm-module if local .m2/repository/ is empty
+* STUDIO-4543 - Rest API Extension validation fails with NPE when project folder and artifactId does not match
+* STUDIO-4547 - Git repository clone failed when migrating a project which has already been cloned in the workspace.
+* STUDIO-4548 - After migration RestApi Extension failed with "org.bonitasoft.engine.bdm.BusinessObjectDaoCreationException: java.lang.ClassNotFoundException: com.md.model.WOMSOrderDAOImpl"
```

Comment:
```suggestion
* STUDIO-4548 - After migration RestApi Extension fails with "org.bonitasoft.engine.bdm.BusinessObjectDaoCreationException: java.lang.ClassNotFoundException: com.md.model.WOMSOrderDAOImpl"
```

Reply:
Would be good to update the ticket titles as well :-)